helm/ovn-kubernetes/charts/ovnkube-single-node-zone/values.yaml

```
@@ -6,4 +6,5 @@ ovnControllerLogLevel: "-vconsole:info"
 logFileMaxSize: 100
 logFileMaxBackups: 5
 logFileMaxAge: 5
-libovsdbClientLogFile: ""
\ No newline at end of file
+libovsdbClientLogFile: ""
+AdvertisedUDNIsolationMode: "strict"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)